src/cms/data/data.ts

```diff
@@ -148,25 +147,27 @@ export async function getRecords(
   return { data: d1Data, source: "d1", total };
 }
 
-export async function insertRecord(d1, kv, data) {
+export async function insertRecord(d1, kv, data, skipD1 = false) {
   const content = data;
   const id = uuidv4();
   const timestamp = new Date().getTime();
   content.data.id = id;
   let error = "";
 
   // console.log("insertRecord", content);
-
+  let result = {};
   try {
-    const result = await saveKVData(kv, id, content.data);
+    result = await saveKVData(kv, id, content.data);
     // console.log('result KV', result);
     // return ctx.json(id, 201);
   } catch (error) {
     error = "error posting content" + error;
   } finally {
     //then also save the content to sqlite for filtering, sorting, etc
     try {
-      const result = await insertD1Data(d1, kv, content.table, content.data);
+      if (!skipD1) {
```

Comment:
D1 is the system of record, so we'll always want to store any data in it. KV and Cache are temporary